servers/quarkus-server/pom.xml

```diff
@@ -327,5 +327,20 @@
         <quarkus.container-image.build>true</quarkus.container-image.build>
       </properties>
     </profile>
+    <profile>
+      <!-- Use the k8s profile to generate manifest files for K8s & Minikube, which will be located under target/kubernetes -->
```

Comment:
Wonder whether we can add a jar-artifact to the deployment (e.g. `nessie-quarkus-1.2.3-k8s.jar`) that contains the generated manifest files. Would that be useful for user?

Reply:
atm I don't think we need to ship these k8s artifacts at all. The way to install Nessie would be only via the Helm chart. If we decide that we want to ship those k8s artifacts at some point, then we would need to adjust a bunch of k8s settings in ´application.properties` but I don't see this in the scope of this PR.